tests/Components.test.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Grid from '../src/game/Grid';
+import RemainingList from '../src/game/RemainingList';
+import Box from '../src/game/Box';
+import RemainingBox from '../src/game/RemainingBox';
+import GridBox from '../src/game/GridBox';
+
+import renderer from 'react-test-renderer';
+
+describe('Components tests', () => {
```

Comment:
All those tests are useless, of course they are crashing when they don't have their required props !